include/swift/SIL/MemAccessUtils.h

```diff
@@ -1536,6 +1536,8 @@ inline Operand *getAccessProjectionOperand(SingleValueInstruction *svi) {
   // would not be casts, but will likely be eliminated with opaque values.
   case SILInstructionKind::OpenExistentialAddrInst:
   case SILInstructionKind::UncheckedTakeEnumDataAddrInst:
+  // We look through mark must check.
+  case SILInstructionKind::MarkMustCheckInst:
```

Comment:
Please also add this to `AddressUseDefWalker.walkUpDefault` and `AddressDefUseWalker.walkDownDefault` in `WalkUtils.swift` (and a test in `SILOptimizer/accessutils.sil`)

Reply:
@eeckstein https://github.com/apple/swift/pull/60989